src/OSPSuite.Core/Domain/Services/SimModelBatch.cs

```diff
@@ -88,6 +88,7 @@ public double[] SensitivityValuesFor(string outputPath, string parameterPath)
       public void Clear()
       {
          _modelCoreSimulation = null;
+         _simModelSimulation?.Dispose();
```

Comment:
@msevestre memory leaks for Pop simulations still present. Is `SimModelBatch.Clear()` called in PK-sim after pop calculations?